accepted/0000-hub_retail.md

```diff
@@ -0,0 +1,213 @@
+- Feature Name: Hub Retail Extension
+- Start Date: 2020-11-30
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This RFC describes an extension of retail features for Hub environment.
+
+# Motivation
+[motivation]: #motivation
+
+Reasons for use of Hub with Retail environment
+----------------------------------------------
+
+- scalability
+  - support large environments with the number of terminal exceeding limits of one SUMA server
+
+- disconnected operation
+  - Peripheral server must stay fully functional when the Hub connection is down
+
+
+Possible configurations
+-----------------------
+
+1. Full
+```
+    Terminal -> Branch Server -> Peripheral Server -> Hub
+```
+
+2. Combined SUMA and Branch server 
+```
+    Terminal -> Branch + Peripheral Server Combined -> Hub
+```
+
+3. Standard
+```
+    Terminal -> Branch Server -> SUMA Server
+```
+
+4. Single server
+```
+    Terminal -> Branch + SUMA Server Combined
+```
+
+
+Global Retail data
+------------------
+
+The following items are typically shared in whole organization, so they should be visible and 
+manageable via Hub.
+
+  - Images - distributed as large tarballs via http
+  - Image metadata - pillars, list of packages
+  - HWTYPE configurations - group formula
+  
+Local data
+----------
+
+The following items do not need central management and can stay on Peripheral Server.
+
+  - Branch groups -  groups containing Branch Server and connected terminals
```

Comment:
IIUC those are regular system groups. They can be created on a peripheral server from the HUB user the uyuni configuration formula. https://github.com/SUSE/salt-formulas/tree/master/uyuni-config-formula

Reply:
Yes, regular groups. Purpose is to help with navigating large number of terminals. Also they are mandatory for successful onboarding of minion (existence is checked during minion registration)